### DISCUSSION TITLE
[要望] ユーザーのミュート機能について、新着の質問も非表示にしてほしい

### DISCUSSION THREAD

**Discussion by blue32a** [increments/qiita-discussions#462] [要望・アイデア・不具合]
現状、ユーザーをミュートにすることで非表示になるコンテンツとして、質問の新着ページは対象外となっています。（トレンド、タイムライン、タグ、回答募集中が非表示となるベージ）

https://help.qiita.com/ja/articles/qiita-mute

しかし”ユーザー個人をミュートにしても新着は見たい”ということは無いので、新着も同様に非表示になると嬉しいです。
もし新着は対象外とすることに何らかの意図があるのであれば、質問を個別にミュートする機能でも良いと思います。

ご検討いただければ幸いです。

**Comment by wataru86** [comment]
@blue32a
ご意見ありがとうございます。ご不便をおかけして申し訳ありません🙏 

こちらの問題は把握しておりましたが、直近の開発優先度などから着手はできていませんでした。
もうしばらくお待ちいただけると幸いです🙇

**Reply by blue32a** [reply]
回答ありがとうございます。
新着ページを中心に見ているので、他と同様に対応されるとありがたいです。
どうぞよろしくお願いいたします。

**Comment by kimkim0814** [comment]
@blue32a
お待たせいたしました!

ユーザーをミュートした際に、新着の質問も非表示になるように今年の6月に対応しておりましたが、
返信の方が漏れておりました🙇‍♂️
ご報告が遅くなってしまい申し訳ありません。

この度はご意見をいただきありがとうございました。
